notebooks/pinnacle/ExportSample.ipynb

```diff
@@ -0,0 +1,235 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pinnacle Export to DICOM Example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import libraries requried"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import tarfile\n",
+    "import shutil\n",
+    "\n",
+    "from pymedphys_pinnacle import Pinnacle"
```

Comment:
Expose this object through the top level `pymedphys` package and then get users to simply run `from PyMedPhys import Pinnacle`